admin/model/payment/razorpay.php

```diff
@@ -496,54 +496,53 @@ public function getSingleSubscription($entity_id)
         return $query->row;
     }
 
-    public function addRecurring($data)
+    public function addSubscription($data)
     {
-        $this->rzpPdo->prepare("INSERT INTO `" . DB_PREFIX . "recurring` SET `status` = :status, `price` = :price, `frequency` = :frequency, `duration` = :duration, `cycle` = :cycle, `trial_status` = :trial_status, `trial_price` = :trial_price, `trial_frequency` = :trial_frequency, `trial_duration` = :trial_duration, `trial_cycle` = :trial_cycle");
-        $this->rzpPdo->bindParam(':status', (int)$data['status']);
-        $this->rzpPdo->bindParam(':price', (float)$data['price']);
-        $this->rzpPdo->bindParam(':frequency', $this->db->escape($data['frequency']));
-        $this->rzpPdo->bindParam(':duration', (int)$data['duration']);
-        $this->rzpPdo->bindParam(':cycle', (int)$data['cycle']);
-        $this->rzpPdo->bindParam(':trial_status', (int)$data['trial_status']);
-        $this->rzpPdo->bindParam(':trial_price', (float)$data['trial_price']);
+        $this->rzpPdo->prepare("INSERT INTO `" . DB_PREFIX . "subscription_plan` SET `trial_frequency` = :trial_frequency, `trial_duration` = :trial_duration, `trial_cycle` = :trial_cycle, `trial_status` = :trial_status, `frequency` = :frequency, `duration` = :duration, `cycle` = :cycle, `status` = :status");

```

Comment:
@abdulwahidsharief , please create new variable for query and use proper line for each columns.